projects/swimlane/ngx-ui/src/lib/components/select/select-dropdown.component.ts

```diff
@@ -200,18 +175,52 @@ export class SelectDropdownComponent implements AfterViewInit {
     this.keyup.emit({ event, value });
   }
 
+  onOptionClick(option: SelectDropdownOption) {
+    this.selection.emit(option);
+  }
+
   onOptionKeyDown(event: KeyboardEvent, option?: SelectDropdownOption): void {
     event.preventDefault();
     event.stopPropagation();
 
-    const key = event.key;
+    switch (event.code) {
+      case KeyboardKeys.ESCAPE:
+        return this.close.emit(true);
+      case KeyboardKeys.ARROW_DOWN:
+        return this.focusNext();
+      case KeyboardKeys.ARROW_UP:
+        return this.focusPrev();
+      case KeyboardKeys.ENTER:
+      case KeyboardKeys.SPACE:
```

Comment:
Does this affect the filter input?

Reply:
Should not... the input has a different event listeners (now named `onInputKeyUp` and `onInputKeyDown`)

Reply:
Oh.. that's the other component... but same answer.  Doesn't impact AFAICT.